tests/syntax_tests/data/printer/expr/expected/asyncAwait.res.txt

```diff
@@ -88,7 +88,7 @@ let _ = await (assert(x))
 let _ = await promises[0]
 let _ = await promises["resolved"]
 let _ = await promises["resolved"] = sideEffect()
-let _ = @attr await (expr)
+let _ = @outer await (@inner expr)
```

Comment:
Update test to showcase the difference between attributes outside and inside the await expression.
This distinction was not possible before adding an ast node for await.